apps/worldclock/app.js

```diff
@@ -118,13 +113,12 @@ function draw() {
   // Loop through offset(s) and render
   offsets.forEach((offset, index) => {
     const dx = getCurrentTimeFromOffset(gmt, offset[OFFSET_HOURS]);
-    hours = doublenum(dx.getHours());
-    minutes = doublenum(dx.getMinutes());
+    var time = require("locale").time(dx, 1);
 
     if (offsets.length === 1) {
       // For a single secondary timezone, draw it bigger and drop time zone to second line
       g.setFont(font, secondaryTimeFontSize);
-      g.drawString(`${hours}:${minutes}`, xyCenter, yposTime2, true);
+      g.drawString(`${time}`, xyCenter, yposTime2, true);
```

Comment:
This and below can be like so, to save a bit of processing:
```suggestion
      g.drawString(time, xyCenter, yposTime2, true);
```